src/dg/strong_dg.cpp

```diff
@@ -212,6 +212,7 @@ void DGStrong<dim,nstate,real,MeshType>::assemble_volume_term_derivatives(
     std::vector< ADArrayTensor1 > conv_phys_flux_at_q(n_quad_pts);
     std::vector< ADArrayTensor1 > diss_phys_flux_at_q(n_quad_pts);
     std::vector< ADArray > source_at_q(n_quad_pts);
+    std::vector< ADArray > physical_source_at_q(n_quad_pts);
```

Comment:
Declare but only allocate if `this->pde_physics_double->has_nonzero_physical_source`

Reply:
fixed.
It is now only declared and allocate (resize) when has_nonzero_physical_source is true.

Reply:
The same work was done for the source_at_q.